library/translation.inc.php

```diff
@@ -263,6 +263,15 @@ function html_header_show() {
     //  We commented this out since we are now standardizing encoding
     //  in the globals.php file.
```

Comment:
We had replaced an old function in translations.inc.php
While not an ideal location, the code may stay here until a complete asset management solution will be introduced.